src/main/haskell/kore/README.md

```diff
@@ -18,6 +18,9 @@ To regenerate the golden data for regression tests:
 `stack test --no-keep-going --ta --accept`
 
 To generate documentation: `stack build --haddock`.
+Note that because of `co-log`'s `typerep-map` dependency (which fails haddock),
+we are currently building with `--no-haddock-deps`. This should be reverted
+once `typerep-map`'s haddock is fixed.
```

Comment:
Could you open an issue on our tracker for this, and include a link to the upstream issue?

Reply:
I have, and they closed it and linked to this bug in `stack`: https://github.com/commercialhaskell/stack/issues/3989 ; I'll add this to the docs here.